src/world/area_mgm/mgm_02/mgm_02_2_npc.c

```diff
@@ -2064,7 +2064,7 @@ StaticNpc N(NpcData_Fuzzy_05) = {
 };
 
 StaticNpc N(NpcData_Bombomb_01) = {
-    .id = NPC_Bombomb_01,
+    .id = NPC_Bobomb_01,
```

Comment:
Looks like the I'd name got fixed here, but not the name of the static NPC struct instance